.semaphore/semaphore.yml

```diff
@@ -5,14 +5,50 @@ agent:
     type: e1-standard-4
     os_image: ubuntu1804
 blocks:
-  - name: 'Testing'
-    task:
-      jobs:
-      - name: 'K8ST (make k8s-test)'
-        commands:
-          # Checkout the code and run the Kubernetes tests.
-          - checkout
-          # Semaphore is doing shallow clone on a commit without tags.
-          # unshallow it for GIT_VERSION:=$(shell git describe --tags --dirty --always) @ Makefile.common
-          - git fetch --unshallow
-          - make k8s-test
+
+# Run the full set of tests. Each job can potentially be run in parallel,
+# provided Semaphore has enough boxes available.
+- name: "Tests"
+  dependencies: []
+  task:
+    prologue:
+      commands:
+      - checkout
```

Comment:
Since `checkout` is common for all blocks, may be use[ global_job_config](https://docs.semaphoreci.com/reference/pipeline-yaml-reference/#global_job_config)

Reply:
Neat, didn't spot that. Much nicer!